### ISSUE TITLE
libstdc++ issue when not on latest version from ubuntu-toolchain-r/test ppa

### ISSUE THREAD

**Issue by daniel-j-h** [Project-OSRM/node-osrm#234]
The readme currently says:

> If you are running Ubuntu older than 14.04 you can easily upgrade your g++ toolchain like:

> sudo add-apt-repository ppa:ubuntu-toolchain-r/test
> sudo apt-get update
> sudo apt-get install g++-4.8

I'm on Ubuntu 15.04 and running `npm install; make test` results in:

```
*** Using osrm-extract from ../../lib/binding/osrm-extract ***
Running osrm-extract...
osrm-extract: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by osrm-extract)
```

The stdlib version installed is `libstdc++6`.

```bash
daniel@x1c ~> strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 |ag glibcxx
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_3.4.14
GLIBCXX_3.4.15
GLIBCXX_3.4.16
GLIBCXX_3.4.17
GLIBCXX_3.4.18
GLIBCXX_3.4.19
GLIBCXX_3.4.20
GLIBCXX_DEBUG_MESSAGE_LENGTH
```

cc @springmeyer @TheMarex

**Comment by springmeyer** [comment]
Addressed in #233

**Comment by springmeyer** [comment]
Closing - #233 is merged.
